packages/collector/test/tracing/database/prisma/test.js

```diff
@@ -68,15 +68,6 @@ mochaSuiteFn('tracing/prisma', function () {
       });
 
       after(async () => {
-        // eslint-disable-next-line no-use-before-define
```

Comment:
https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/connection-management

> In most cases, you do not need to explicitly call these methods. PrismaClient automatically connects when you run your first query, creates a [connection pool](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/connection-pool), and disconnects when the Node.js process ends.


Look what I found. Prisma automatically disconnects when the process get's killed.

> None of the other DB integration test apps has such a manual disconnect step

True, but I think they should. I think disconnecting from an external resource before the process get's killed is correct. But nothing we should actively change now

Reply:
Good find.

Reply:
About disconnecting from the DB from the apps for other integration tests: It might not matter. The app under test is killed anyway. Worst case is that the DB keeps the connection open for a while longer. Most DBs will probably close the connection on their end after a while when a client terminates without disconnecting first. And we also stop the DB containers once all tests are done.